src/main/resources/db/migration/V2__next_migration.sql

```diff
@@ -0,0 +1,7 @@
+--V2__next_migration.sql
+CREATE TABLE auth (
+    id SERIAL PRIMARY KEY,
+    email VARCHAR(255) NOT NULL UNIQUE,
+    password VARCHAR(255) NOT NULL,
+    isActive BOOLEAN DEFAULT FALSE
```

Comment:
database table name as user_authentication.
We should use 'is_active' for column name instead of 'isActive'.
It is also a good practice to add 'created_at', 'updated_at' column.

Reply:
resolved